devlib/utils/types.py

```diff
@@ -68,6 +68,15 @@ def numeric(value):
     """
     if isinstance(value, int):
         return value
+
+    if isinstance(value, basestring):
+        value = value.strip()
+        if value.endswith('%'):
+            try:
+                return float(value.rstrip('%')) / 100
```

Comment:
Well, you'll get a list of integers which is a not too bad behaviour IMO, but I agree you have a problem if you call it with `"['a', 'b']"`. Fair enough, `ast.eval_literal()` is maybe too powerful to reduce really simplify the code in any way ...